src/test/ui/feature-gate/await-macro.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0658]: `await!(<expr>)` macro syntax is unstable, and will soon be removed in favor of `<expr>.await` syntax.
```

Comment:
Heads up: You'll need to rebase because this feature gate is gone.

Reply:
Will do